<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detectWindow</class>
 <widget class="QMainWindow" name="detectWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chemistry Detection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/resource/10125_icons/shell32_35.ico</normaloff>:/resource/10125_icons/shell32_35.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainwindowVLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionExport_Pdf"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAdvanced"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionFullscreen"/>
    <addaction name="actionRestore"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Original"/>
    <addaction name="actionZoom_Fit_Best"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt_Creator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="menuToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Window"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExport_Pdf"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionFullscreen"/>
   <addaction name="actionRestore"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_Fit_Best"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/newwindow.ico</normaloff>:/resource/mine/newwindow.ico</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/openfile.ico</normaloff>:/resource/mine/openfile.ico</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/save.ico</normaloff>:/resource/mine/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/saveas.ico</normaloff>:/resource/mine/saveas.ico</iconset>
   </property>
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/print.ico</normaloff>:/resource/mine/print.ico</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset theme="document-print-preview">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/10125_icons/imageres_5315.ico</normaloff>:/resource/10125_icons/imageres_5315.ico</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/10125_icons/imageres_5311.ico</normaloff>:/resource/10125_icons/imageres_5311.ico</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/10125_icons/shell32_16762.ico</normaloff>:/resource/10125_icons/shell32_16762.ico</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/10125_icons/shell32_243.ico</normaloff>:/resource/10125_icons/shell32_243.ico</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/10125_icons/shell32_16763.ico</normaloff>:/resource/10125_icons/shell32_16763.ico</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/delete.ico</normaloff>:/resource/mine/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAdvanced">
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset theme="edit-find-replace">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Find/Replace</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout_Qt_Creator">
   <property name="text">
    <string>About Qt Creator...</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/refresh.ico</normaloff>:/resource/mine/refresh.ico</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/fullscreen.ico</normaloff>:/resource/mine/fullscreen.ico</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/restore.ico</normaloff>:/resource/mine/restore.ico</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/zoomin.ico</normaloff>:/resource/mine/zoomin.ico</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/zoomout.ico</normaloff>:/resource/mine/zoomout.ico</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_Original">
   <property name="text">
    <string>Zoom Original</string>
   </property>
  </action>
  <action name="actionZoom_Fit_Best">
   <property name="icon">
    <iconset theme="zoom-fit-best">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit Best</string>
   </property>
  </action>
  <action name="actionGoNext">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/gonext.ico</normaloff>:/resource/mine/gonext.ico</iconset>
   </property>
   <property name="text">
    <string>GoNext</string>
   </property>
  </action>
  <action name="actionGoPrevious">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/goprevious.ico</normaloff>:/resource/mine/goprevious.ico</iconset>
   </property>
   <property name="text">
    <string>GoPrevious</string>
   </property>
  </action>
  <action name="actionExport_Pdf">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/resource/mine/exportpdf.ico</normaloff>:/resource/mine/exportpdf.ico</iconset>
   </property>
   <property name="text">
    <string>Export Pdf</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
